packages/eas-cli/src/metadata/context.ts

```diff
@@ -0,0 +1,110 @@
+import { App, Session } from '@expo/apple-utils';
+import { ExpoConfig } from '@expo/config';
+import { Platform } from '@expo/eas-build-job';
+import { IosSubmitProfile } from '@expo/eas-json/build/submit/types';
+import assert from 'assert';
```

Comment:
I'm not sure why it's not enforced by eslint (it should be) but the convention in Expo projects is usally:

```
// node built-in modules imports

// 3rd party modules imports

// local imports
```

There should be an empty line between each group.

Reply:
I thought the convention was:

```
// Node or 3rd party modules

// Local imports
```

Just like the **[project/expoSdk.ts](https://github.com/expo/eas-cli/blob/07d6b5269bc1bf4b18cd2cd5d5a3342603c8218d/packages/eas-cli/src/project/expoSdk.ts#L1-L8)**.

Reply:
Ok, I thought we follow this convention in EAS CLI too. 

In turtle-v2 repo we certainly have this rule enforced. See e.g. https://github.com/expo/turtle-v2/blob/main/src/services/submission-service/src/tasks/iosSubmit/ascApiKey.ts#L1 

When I change the order I get an error from linter
<img width="837" alt="Screenshot 2022-06-02 at 15 58 21" src="https://user-images.githubusercontent.com/5256730/171646979-37b5467b-d876-45b2-85c1-8f0e4cc7087f.png">